packages/manager/apps/pci-project/src/pages/home/Header.page.tsx

```diff
@@ -0,0 +1,66 @@
+import { Outlet } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { useProject } from '@ovh-ux/manager-pci-common';
+import {
+  // OdsSpinner,
+  OdsText,
+  OdsBadge,
+  OdsBreadcrumb,
+  OdsBreadcrumbItem,
+} from '@ovhcloud/ods-components/react';
+import {
+  BaseLayout,
+  ChangelogButton,
+  Title,
+  useProjectUrl,
+} from '@ovh-ux/manager-react-components';
+import Navigation from './Navigation.page';
+import NotFound from '@/pages/404';
+import { urls } from '@/routes/routes.constant';
+
+export default function ProjectHeader() {
+  const { t } = useTranslation(['common', 'home', 'settings']);
+
+  const { data: project, isError } = useProject();
+
+  const appHomeUrl = useProjectUrl('public-cloud') + urls.home;
+
+  if (!project) return <NotFound />;
+  if (isError) return <OdsText>{t('error')}</OdsText>;
```

Comment:
I think, in case of error, you can let it crash, the ErrorBoundary will catch it and display an error